README.md

```diff
@@ -1,7 +1,94 @@
-# Stoat
 
-Turn PR comments into developer dashboards with templating and artifact previews for code coverage reports, frontend components, doc sites, and more! [Visit our docs website to learn more ↗︎](https://docs.stoat.dev/)
+<h1 align="center">
+  <br>
+  <a href="https://docs.stoat.dev/"><img src="https://stoat-dev--static.stoat.page/branding/android-chrome-192x192.png" alt="Stoat" width="192"></a>
+  <br>
+  Stoat
+  <br>
+</h1>
 
----
+<h4 align="center">Turn pull request comments into developer dashboards.</h4>
 
-![comment screenshot](docs/static/img/example-screenshot.png)
+<p align="center">
+  <a href="https://github.com/stoat-dev/stoat-action/actions/workflows/test-action.yaml">
+    <img src="https://img.shields.io/github/actions/workflow/status/stoat-dev/stoat-action/test-action.yaml?branch=main&label=action"
+         alt="action build status">
+  </a>
+  <a href="https://github.com/stoat-dev/stoat-action/actions/workflows/test-cli.yaml">
+    <img src="https://img.shields.io/github/actions/workflow/status/stoat-dev/stoat-action/test-cli.yaml?branch=main&label=cli"
+         alt="cli build status">
+  </a>
+  <a href="https://github.com/stoat-dev/stoat-action/actions/workflows/docs.yaml">
+    <img src="https://img.shields.io/github/actions/workflow/status/stoat-dev/stoat-action/docs.yaml?branch=main&label=docs"
+         alt="docs build status">
+  </a>
+</p>
+
+<p align="center">
+  <a href="https://twitter.com/stoat_dev">
+    <img src="https://img.shields.io/twitter/follow/stoat_dev?style=social"
+         alt="twitter">
+  </a>
+  <a href="https://github.com/stoat-dev/stoat-action">
+    <img src="https://img.shields.io/github/stars/stoat-dev/stoat-action?style=social"
+         alt="github stars">
+  </a>
+</p>
+
+<p align="center">
+  <a href="#quick-start">Quick Start</a> •
+  <a href="#how-to-use-stoat">How to Use Stoat</a> •
+  <a href="#license">License</a>
+</p>
+
+<p align="center">
+<img src="https://stoat-dev--static.stoat.page/screenshot-python.png" alt="screenshot" width="600">
```

Comment:
Nitpick.

This runtime chart only has two data points. It is not immediately clear what it is about, mainly because on the x-axis the two commits are at the ends, and not easy to see.

What about using the chart from https://github.com/stoat-dev/stoat-action/pull/77#issuecomment-1354195832

The static hosting section can remain the same, because the current screenshot has more entries.

Reply:
I'll leave as is for now but then add a better screenshot once we have the default branch's data prefixed to it. I think that will be the most realistic: one or two branch dots after the 10 most recent commits.